python/protoc-gen-mypy

```diff
@@ -160,11 +160,12 @@ class PkgWriter(object):
                     for field in [f for f in desc.field if f.label == d.FieldDescriptorProto.LABEL_REQUIRED]:
                         l("{} : {},", field.name, self.python_type(field))
                     for field in [f for f in desc.field if f.label != d.FieldDescriptorProto.LABEL_REQUIRED]:
+                        self._import("typing", "Optional")
```

Comment:
can you please do this as `optional = self._import("typing", "Optional")` and use `optional` var below? This matters because if there is an `Optional` message in the proto the import will be aliased under a different name. thanks!